internal/commands/import_test.go

```diff
@@ -0,0 +1,59 @@
+//go:build !integration
+
+package commands
+
+import (
+	"testing"
+
+	errorconstants "github.com/checkmarx/ast-cli/internal/constants/errors"
+	"github.com/checkmarx/ast-cli/internal/wrappers/mock"
+	"gotest.tools/assert"
+)
+
+func TestImport_ImportSarifFileWithCorrectFlags_CreateImportSuccessfully(t *testing.T) {
+	execCmdNilAssertion(t, "import", "--project-name", "my-project", "--import-file-path", "my-path.sarif")
+}
+
+func TestImport_ImportSarifFileProjectDoesntExist_CreateImportWithProvidedNewNameSuccessfully(t *testing.T) {
+	execCmdNilAssertion(t, "import", "--project-name", "MOCK-PROJECT-NOT-EXIST", "--import-file-path", "my-path.sarif")
+}
+
+func TestImport_ImportSarifFileMissingImportFilePath_CreateImportReturnsErrorWithCorrectMessage(t *testing.T) {
+	err := execCmdNotNilAssertion(t, "import", "--project-name", "my-project", "--import-file-path", "")
+	assert.Assert(t, err.Error() == errorconstants.ImportFilePathIsRequired)
+}
+
+func TestImport_ImportSarifFileMissingImportProjectName_CreateImportReturnsErrorWithCorrectMessage(t *testing.T) {
+	err := execCmdNotNilAssertion(t, "import", "--import-file-path", "my-path.zip")
+	assert.Assert(t, err.Error() == errorconstants.ProjectNameIsRequired)
+}
+
+func TestImport_ImportSarifFileProjectNameNotProvided_CreateImportWithProvidedNewNameSuccessfully(t *testing.T) {
+	err := execCmdNotNilAssertion(t, "import", "--project-name", "", "--import-file-path", "my-path.sarif")
+	assert.Assert(t, err.Error() == errorconstants.ProjectNameIsRequired)
+}
+
+func TestImport_ImportSarifFileUnacceptedFileExtension_CreateImportReturnsErrorWithCorrectMessage(t *testing.T) {
+	err := execCmdNotNilAssertion(t, "import", "--project-name", "MOCK-PROJECT-NOT-EXIST", "--import-file-path", "my-path.txt")
+	assert.Assert(t, err.Error() == errorconstants.SarifInvalidFileExtension)
+}
+
+func TestImport_ImportSarifFileMissingExtension_CreateImportReturnsErrorWithCorrectMessage(t *testing.T) {
+	err := execCmdNotNilAssertion(t, "import", "--project-name", "MOCK-PROJECT-NOT-EXIST", "--import-file-path", "some/path/no/extension/my-path")
+	assert.Assert(t, err.Error() == errorconstants.SarifInvalidFileExtension)
+}
+
```

Comment:
The following tests for "importFile" were created to test the mapping of the http error codes to user-friendly error codes.
When we will be able to define an expectation of mocks we'll be able to properly test the wrappers by mocking client.go